index.html

```diff
@@ -1,234 +1,216 @@
-                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-github" src="images/social/Linkedin.png" alt="github"></a></li>
-                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-thumb" src="images/social/Thumb-Enabled.png" alt="thumb"></a></li>
-                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-twitter" src="images/social/Twitter.png" alt="twitter"></a></li>   
-                <li><a href="https://github.com/alexisuwimana" class=""><img class="social-M" src="images/social/M-Disabled.png" alt="M"></a></li>   
-            </ul>
+    <div class="overall-container">
+      <section class="main-container">
+        <div class="intro-header">
+          <h1>
+            Hey There. <br />
+            I’m Alexis
+          </h1>
+          <div class="intro-detail">
+            <h2 class="intro-title">I’am a Software Developer</h2>
+            <p>
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
         </div>
-    </section> 
-
-        <!-- works -->
-
-    <section class="Works">
-        <div class="works-title" >
-            <h3 class="heading-3">My Recent Works </h3> 
-            <hr class="works-line">
+        <div class="social">
+          <ul class="social-ul">
+            <li>
+              <a href="#"><i class="fab fa-github" title="github icon"></i></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-linkedin-in" title="linkedin icon"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-angellist" title="Angel icon"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-twitter" title="twitter icon"></i
+              ></a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-medium-m" title="Medium icon"></i
+              ></a>
+            </li>
+          </ul>
         </div>
-        <div class="works-img">
-            <img src="./images/works/img-works-0.svg" alt="works">
+      </section>
+      <section class="my-projects-section">
+        <div id="projects-title">
+          <h1>My Recent Works</h1>
         </div>
-        <div class="stories">
-            <h1 class="stories-headings">Multi-Post Stories</h1>
-            <p class="stories-content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="inner-job-item-two">
         </div>
```

Comment:
- [ ] Great job implementing your regular projects using JavaScript. However, there are some style issues that you need to fix to get the same design as the template. `Requirement:  When the project is hovered on with the mouse the description and dark opacity disappears and a See Project button appears to the bottom of the image.`
- First, in your regular projects, the button doesn't appear at the bottom of the card.
- It would be better if the description and dark opacity disappears when the project is hovered.

![Regular projects](https://user-images.githubusercontent.com/79876959/139561273-0a3c5ffe-0704-44bb-8a9f-79cd7835d9b9.PNG)

**Figma Template**

![Regular projects template](https://user-images.githubusercontent.com/79876959/139561220-95a0fb7e-0067-473c-b06c-0ff0c3af1aa7.PNG)